.github/workflows/phpunit.yml

```
@@ -17,7 +17,7 @@ jobs:
         os: [ubuntu-18.04]
         epubcheck: [4.2.4]
         prince: [12.5-1]
-        wordpress: [5.6.2]
+        wordpress: [5.6.2, 5.7.0]
```

@SteelWagstaff The reason this isn't working is that WordPress "major" versions aren't versioned using "real" SemVer (they omit the final `.0`). This should be:

```suggestion
        wordpress: [5.6.2, 5.7]
```